services/web/server/src/simcore_service_webserver/projects/projects_handlers.py

```diff
@@ -406,6 +406,31 @@ async def replace_project(request: web.Request):
         if current_project["accessRights"] != new_project["accessRights"]:
             await check_permission(request, "project.access_rights.update")
 
+        if await director_v2_api.is_pipeline_running(
```

Comment:
So you actually prevent updating the study while the pipeline is running. 
Could this be fine grained to "prevent saving if you try to update links"?

Reply:
I implemented a granula check.
Please review again that the conditions fit what we discussed.
thx